pkg/context/context.go

```diff
@@ -38,6 +38,7 @@ type GlobalVars struct {
 	CaptureInfo *model.CaptureInfo
 	EtcdClient  *etcd.CDCEtcdClient
 	GrpcPool    kv.GrpcPool
+	PDPhyTs     int64
```

Comment:
It may cause data race, because global var may be accessed in different goroutines.

Could you use `*PDTimeCache` instead? (And `*PDTimeCache` must be thread-safe.)

```suggestion
	PDTime *PDTimeCache
```

Reply:
good advice~ thanks